datalake/src/main/java/com/sequenceiq/datalake/service/upgrade/SdxRuntimeUpgradeService.java

```diff
@@ -230,13 +229,7 @@ private String validateImageId(List<ImageInfoV4Response> upgradeCandidates, Stri
     private SdxUpgradeReplaceVms shouldReplaceVmsAfterUpgrade(SdxUpgradeRequest upgradeRequest) {
         return Optional.ofNullable(upgradeRequest)
                 .map(SdxUpgradeRequest::getReplaceVms)
-                .orElse(NullUtil.getIfNotNullOtherwise(upgradeRequest, this::determineReplaceVmsIfParamIsMissing, SdxUpgradeReplaceVms.DISABLED));
-    }
-
-    private SdxUpgradeReplaceVms determineReplaceVmsIfParamIsMissing(SdxUpgradeRequest request) {
-        SdxUpgradeReplaceVms replaceVms = isOsUpgrade(request) ? SdxUpgradeReplaceVms.ENABLED : SdxUpgradeReplaceVms.DISABLED;
-        LOGGER.debug("VM-s replacement after the upgrade process is {}", replaceVms.name());
-        return replaceVms;
+                .orElse(SdxUpgradeReplaceVms.ENABLED);
```

Comment:
Could we pls keep the log? It could be useful: LOGGER.debug("VM-s replacement after the upgrade process is {}", replaceVms.name());